ocf_data_sampler/torch_datasets/utils/config_normalization_values_to_dicts.py

```diff
@@ -40,18 +41,34 @@ def config_normalization_values_to_dicts(
         stds_dict["nwp"] = {}
 
         for nwp_key in config.input_data.nwp:
-            # Standardise and convert to NumpyBatch
+            nwp_config = config.input_data.nwp[nwp_key]
 
-            means_dict["nwp"][nwp_key] = channel_dict_to_dataarray(
-                config.input_data.nwp[nwp_key].channel_means,
-            )
-            stds_dict["nwp"][nwp_key] = channel_dict_to_dataarray(
-                config.input_data.nwp[nwp_key].channel_stds,
-            )
+            means_list = []
+            stds_list = []
+
+            for channel in list(nwp_config.channels):
+                # These accumulated channels are diffed and renamed
+                if channel in nwp_config.accum_channels:
+                    channel =f"diff_{channel}"
+
+                means_list.append(nwp_config.normalisation_constants[channel].mean)
+                stds_list.append(nwp_config.normalisation_constants[channel].std)
+
+            means_dict["nwp"][nwp_key] = np.array(means_list)
+            stds_dict["nwp"][nwp_key] = np.array(stds_list)
 
     if config.input_data.satellite is not None:
+        sat_config = config.input_data.satellite
+
+        means_list = []
+        stds_list = []
+
+        for channel in list(config.input_data.satellite.channels):
+            means_list.append(sat_config.normalisation_constants[channel].mean)
+            stds_list.append(sat_config.normalisation_constants[channel].std)
 
-        means_dict["sat"] = channel_dict_to_dataarray(config.input_data.satellite.channel_means)
-        stds_dict["sat"] = channel_dict_to_dataarray(config.input_data.satellite.channel_stds)
+        # Convert to array and expand dimensions so we can normalise the 4D sat and NWP sources
+        means_dict["sat"] = np.array(means_list)[None, :, None, None]
+        stds_dict["sat"] = np.array(stds_list)[None, :, None, None]
```

Comment:
IM slightly confusued here, the NWP means and stds look like 1 d arrays, but these look like 4D arrays, perhaps Ive missed something

Reply:
Strange a test didnt find that? Could we add one, or add an issue for it?